### PR TITLE
Defect Fix : Doc how to send Progress file for help

### PR DESCRIPTION
**Pull request by radhikapc**
This is a PR corresponding the issue https://www.pivotaltracker.com/n/projects/1902233/stories/140599415

### CODE REVIEW

**Comment by joshix** [comment]
Almost there. The document must next explain very concisely how to provide the Progress file. Do I attach it to an Issue in this repo? Do I email it? To what address do I email it?

**Comment by joshix** [comment]
@ericchiang Could you give a quick look at this before I merge it, since I believed you worked with @radhikapc on it?

File: README.md [hunk 1]
```diff
@@ -27,3 +29,16 @@ To make the bug report accurate and easy to understand, please try to create iss
 * _Scoped_. One bug per issue; please do not follow up with another bug inside one report. It may be worthwhile to read [Elika Etemad’s article on filing good bug reports](http://fantasai.inkedblade.net/style/talks/filing-good-bugs/) before creating a bug report.
 
 We might ask for further information to locate a bug. A duplicated bug report will be closed.
+
+#### Tectonic Installer Progress file
+
+Click the *Save progress* button at the top right corner of any Installer screen to download a snapshot of install configuration and progress. A JSON-formatted file named `tectonic-progress` will be written beneath the browser's download path. Including the Progress file from a failed installation can assist troubleshooting. Attach the Progress file to the corresponding Issue in this repository.
+
+Before sending a Progress file, ensure that the potentially sensitive values of the following keys are removed or obscured from the Progress file:
+
```
Comment (ericchiang) [inline README.md]: `adminPassword` too, right?

There's just so much sensitive data there. Are we sure we want to do this? cc @sym3tri

Comment (joshix) [inline README.md]: @ericchiang I think this is a reasonable question.

**Comment by joshix** [comment]
Doc is accurate as proposed, but given the difficulty of cleansing sensitive info in these files and the removal of the Progress file facility in a future release, closing for now.

**Pull request closed without merging**